build_files/shared/build-dx.sh

```diff
@@ -28,7 +28,7 @@ sysctl -p
 /ctx/build_files/dx/04-override-install-dx.sh
 
 # Branding Changes
-sed -i '/^PRETTY_NAME/s/Bluefin/Bluefin-dx/' /usr/lib/os-release
+sed -i '/^PRETTY_NAME/s/Bluefin/Bluefin DX/' /usr/lib/os-release
```

Comment:
```suggestion
sed -i '/^PRETTY_NAME/s/Bluefin/Bluefin/' /usr/lib/os-release
```

Reply:
Makes perfect sense to me, let's remove the line entirely since it isn't doing anything anymore :)

Reply:
oh, derp!